### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by galdd**

### CODE REVIEW

**Review by ofir-zeitoun — DISMISSED** [review]
test

File: README.md [hunk 1]
```diff
@@ -1,32 +1,91 @@
-# expanses-client
+
+---
+
+# Expanses Client
 
 # React + TypeScript + Vite
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+```markdown
```
Comment (ofir-zeitoun) [inline README.md:8]: no need for that (this is what breaks the page)

File: README.md [hunk 2]
```diff
@@ -1,32 +1,91 @@
...
+   - [Expense Management Dashboard](#expense-management-dashboard)
+4. [Installation and Setup](#installation-and-setup)
+5. [Running the Application](#running-the-application)
+6. [Key Features](#key-features)
+7. [Technologies Used](#technologies-used)
+8. [Future Implementation](#future-implementation)
+9. [Project Status](#project-status)
+10. [Contributing](#contributing)
+11. [License](#license)
+```
```
Comment (ofir-zeitoun) [inline README.md:23]: same here to remove markdown

**Review by galdd — COMMENTED** [review]
Table of Contents fixed

**Review by ofir-zeitoun — APPROVED** [review]
LGTM

**Pull request merged**